Detectors/ITSMFT/ITS/workflow/src/TrackerSpec.cxx

```diff
@@ -118,6 +120,10 @@ void TrackerDPL::run(ProcessingContext& pc)
         tracks.swap(mTracker->getTracks());
```

Comment:
Idem here: ``o2::ITS::IOUtils::loadROFrameData`` should be modified to use not the mROFrame of the cluster but the ``ROFRecord``.

Reply:
@shahor02 Also, yes.   I will have a look inside (never did before). Hopefully, this place can be modified in the same way as the one above.